hedera-node/hedera-app/src/main/java/com/hedera/node/app/info/MonoNodeInfo.java

```diff
@@ -40,6 +40,6 @@ public MonoNodeInfo(@NonNull com.hedera.node.app.service.mono.context.NodeInfo d
 
     @Override
     public boolean isSelfZeroStake() {
-        return delegate.isSelfZeroStake();
+        return delegate.isSelfZeroWeight();
```

Comment:
Is this really right? In my undestanding, the term 'weight' was introduced to make the difference between 'stake' and 'weight' more explicit, i.e. there must be a difference. It does not look right, if a method 'isSelfZeroStake()' calls a method 'isSelfZeroWeight()'.

The other question is, if this is really what we want. So far we used this check to ensure that a node without stake submits transactions. Is it really correct to replace stake with weight in this check?

Reply:
Changed.